bindata/v4.1.0/kube-apiserver/audit-policies-cm.yaml

```diff
@@ -9,9 +9,6 @@ data:
     kind: Policy
     metadata:
       name: Default
-    # Don't generate audit events for all requests in RequestReceived stage.
-    omitStages:
-    - "RequestReceived"
```

Comment:
we want this omit after the probes, right?

Reply:
I thought we take the union of the `omitStages`

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go#L38-L57